doc/source/release-notes/Percona-Server-5.7.29-32.rst

```diff
@@ -0,0 +1,37 @@
+.. rn:: 5.7.29-32:
+
+================================================================================
+*Percona Server for MySQL* 5.7.29-32
+================================================================================
+
+:Date: February 5, 2020
+
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.29-32
+includes all the features and bug fixes available in
+`MySQL 5.7.29 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-29.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-1469`: The Memory storage engine detected an incorrect "is full" condition when the space contained reusable memory chunks and could be reused.
```

Comment:
reusable memory chunks THAT could be reused

Reply:
Fixed with "that"